mail_message.go

```diff
@@ -122,6 +122,12 @@ func (message *MailMessage) Handle() {
 	}
 
 	if !message.isDebug() {
+		err = message.postToMattermost()
+		if err != nil {
+			err = fmt.Errorf("Error posting message to Mattermost: %v", err)
+			message.handleError(err, len(actualRecipients))
```

Comment:
I'm not sure if we want to do this; this will cause the sender to get a message if this ever fails, which could be confusing to them.